### ISSUE TITLE
Analytics should ignore prepaid conversions

### ISSUE THREAD

**Issue by coenjacobs** [woothemes/woocommerce-google-analytics-integration#7]
This might need some fixing in WooCommerce core as well, because it applies to reports too. Logging it here for further investigating.

> The actual moment of doing the deposit, that should be the order shown in the reports. That is the moment where there is income for the store and not when the customer eventually uses those funds.So in a perfect world, we will need a way to have the reports ignore orders that are paid for by previously deposited funds. Does that make any sense to you? Let's focus on getting something set up in core for this use. I can think of something, but maybe you have ideas? Let's agree that this is the way to go first, before we start thinking on a solution that we don't even need.

Ref: https://woothemes.zendesk.com/agent/#/tickets/86785

**Comment by mikejolley** [comment]
This is related to account funds I assume? @BFTrick thoughts on this?

**Comment by BFTrick** [comment]
Yeah Accounts Funds & Deposits. Something we can look into but this was logged in 2013 and we haven't seen a ton of people require this.

**Comment by justinshreve** [comment]
@mikejolley per the above comment - is this something we still want to look at right now, or wontfix unless we see a ton of other requests?

**Comment by mikejolley** [comment]
I actually remember rewriting account funds to allow partial payment, so this issue may not even be technically possible. Lets leave it.
